src/core/libmaven/isotopeDetection.h

```diff
@@ -26,6 +26,7 @@ class IsotopeDetection
 		bool D2Flag);
 
 	void pullIsotopes(PeakGroup *group);
+	bool filterIsotope(Isotope x, bool C13Flag, bool N15Flag, bool S34Flag, bool D2Flag, float parentPeakIntensity, float isotopePeakIntensity, mzSample* sample, PeakGroup* parentGroup = NULL);
```

Comment:
break this line with no. of character not more than 80

Reply:
@IpankajI Changed in future branches. Most of the parameters have been removed as well.